client/src/views/profile/AuthenticationPage.vue

```diff
@@ -7,13 +7,22 @@
         <authentication-card
           :auth-method="DeviceSaveStrategyTag.Keyring"
           :state="AuthenticationCardState.Active"
+          :hide-checkmark="true"
           v-show="currentDevice && currentDevice.ty.tag === AvailableDeviceTypeTag.Keyring"
         />
         <authentication-card
           :auth-method="DeviceSaveStrategyTag.Password"
           :state="AuthenticationCardState.Active"
+          :hide-checkmark="true"
           v-show="currentDevice && currentDevice.ty.tag === AvailableDeviceTypeTag.Password"
         />
+        <!-- TO DO: uncomment for Smartcard -->
```

Comment:
This one doesn't need to be commented, no device has `SmartCard` as a tag anyway.